.github/workflows/issue-mark-duplicate.yml

```diff
@@ -0,0 +1,17 @@
+name: Issue Mark Duplicate
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  mark-duplicate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: mark-duplicate
+        uses: actions-cool/issues-helper@1.x
+        with:
+          actions: 'mark-duplicate'
+          token: ${{ secrets.GITHUB_TOKEN }}
+          duplicate-labels: 'type:duplicate'
+          close-issue: true
```

Comment:
```suggestion

```

If you don't need to close the issue option, I can remove this.

Reply:
No, I want close it and add label, ONLY if the one have "commit access" send comment.

Reply:
OK, let me update code to realize it.